opendj-server-legacy/pom.xml

```diff
@@ -16,1349 +16,1348 @@
+                            <testExcludes>
+                                <testExclude>**/org/opends/server/snmp/**</testExclude>
+                            </testExcludes>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!-- Disable surefire plugin because all tests are run with failsafe plugin -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-test</id>
+                        <phase>none</phase>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <executions>
+                    <!-- Create opendj-slf4j-adapter manifest -->
+                    <execution>
+                        <id>opendj-slf4j-adapter-manifest</id>
+                        <goals>
+                            <goal>manifest</goal>
+                        </goals>
+                        <configuration>
+                            <buildDirectory>${project.build.directory}/slf4j-adapter-manifest/META-INF</buildDirectory>
+                            <manifestLocation>${project.build.directory}/slf4j-adapter-manifest/META-INF</manifestLocation>
+                            <instructions>
+                                <!-- Export only slf4j adapter -->
+                                <Export-Package>org.opends.server.loggers.slf4j</Export-Package>
+                                <Import-Package>
+                                    org.forgerock.i18n,
+                                    org.forgerock.i18n.slf4j,
+                                    org.opends.messages,
+                                    org.opends.server.loggers,
+                                    org.slf4j,
+                                    org.slf4j.helpers,
+                                    org.slf4j.spi
+                                </Import-Package>
+                            </instructions>
+                        </configuration>
+                    </execution>
+
+                    <!-- Create Wren:DS manifest -->
+                    <execution>
+                        <id>opendj-manifest</id>
+                        <goals>
+                            <goal>manifest</goal>
+                        </goals>
+                        <configuration>
+                            <instructions>
+                                <Bundle-Version>${project.version}</Bundle-Version>
+                                <Export-Package>org.forgerock.opendj.server.embedded</Export-Package>
+                                <Import-Package>
```

Comment:
This needs to be addressed after JEB is not distributed as a separated artifact. I think we should wait for @fyrbach's work before merging this.